articles/remote-rendering/quickstarts/convert-model.md

```diff
@@ -202,11 +202,15 @@ The conversion script generates a *Shared Access Signature (SAS)* URI for the co
 The SAS URI created by the conversion script will only be valid for 24 hours. However, after it expired you do not need to convert your model again. Instead, you can create a new SAS in the portal as described in the next steps:
 
 1. Go to the [Azure portal](https://www.portal.azure.com)
-1. Click on your **Storage account** resource:
+2. Click on your **Storage account** resource:
```

Comment:
Does the numbering break when you insert newlines?

Reply:
> Does the numbering break when you insert newlines?

No. It looks fine in GitHub preview. -jborsecnik